test/inductor/test_torchinductor_opinfo.py

```diff
@@ -273,22 +274,22 @@ def process(device_type):
     "corrcoef": {f16, f32, f64, i32, i64},
     "cov": {f16, f32, f64, i32, i64},
     "equal": {b8, f16, f32, f64, i32, i64},
-    "fft.fft": {f16, f32, f64},
+    "fft.fft": {b8, f16, f32, f64, i32, i64},
```

Comment:
What's the story about these new expected failures?

Reply:
fft support here is extremely flakey. these kernels output complex, so the `requiring_stride_order` on the end breaks when we interact with complex dtypes